Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```
@@ -173,6 +173,31 @@ script:
   - arguments:
     - description: Commit description.
       name: description
+    - description: To commit admin-level changes on a firewall, include the administrator
+        name in the request.
+      name: admin_name
+    - auto: PREDEFINED
+      description: (True/False) Force Commit
```

```suggestion
      description: Force Commit.
```
No need for the True/False in the description.